packages/web/src/components/common/select-price-range-custom/SelectPriceRangeCustom.tsx

```diff
@@ -1,157 +1,261 @@
 
-  const token1Symbol = useMemo(() => {
-    return tokenB?.symbol || "";
-  }, [tokenB]);
+  const comparedTokenA = selectPool.compareToken?.symbol !== tokenB.symbol;
 
-  const tabItems = useMemo(() => {
-    return [token0Symbol, token1Symbol];
-  }, [token0Symbol, token1Symbol]);
+  const currentTokenA = useMemo(() => {
+    return comparedTokenA ? tokenA : tokenB;
+  }, [comparedTokenA, tokenA, tokenB]);
 
-  const currentPriceInfo = useMemo(() => {
-    if (!currentTick) {
+  const currentTokenB = useMemo(() => {
+    return comparedTokenA ? tokenB : tokenA;
+  }, [comparedTokenA, tokenA, tokenB]);
+
+  const currentPriceStr = useMemo(() => {
+    if (!selectPool.currentPrice) {
       return "-";
     }
-    const currentPrice = BigNumber(currentTick.price).toFixed(4);
-    return `${currentPrice} ${token0Symbol} per ${token1Symbol}`;
-  }, [currentTick, token0Symbol, token1Symbol]);
-
-  const findPreviousTick = useCallback((tick: PoolTick) => {
-    const tickIndex = ticks.findIndex(item => item.tick === tick.tick);
-    if (tickIndex < 0) {
-      return undefined;
-    }
-    if (tickIndex < 1) {
-      return ticks[0];
-    }
-    return ticks[tickIndex - 1];
-  }, [ticks]);
+    const currentPrice = toNumberFormat(selectPool.currentPrice, 4);
+    return `${currentPrice} ${currentTokenA.symbol} per ${currentTokenB.symbol}`;
+  }, [currentTokenA.symbol, currentTokenB.symbol, selectPool.currentPrice]);
 
-  const findNextTick = useCallback((tick: PoolTick) => {
-    const tickIndex = ticks.findIndex(item => item.tick === tick.tick);
-    if (tickIndex < 0) {
-      return undefined;
+  const minPriceStr = useMemo(() => {
+    return numberToFormat(`${selectPool.minPrice || 0}`, 4);
+  }, [selectPool.minPrice]);
+
+  const maxPriceStr = useMemo(() => {
+    if (selectPool.selectedFullRange) {
+      return "∞";
     }
-    if (tickIndex >= ticks.length - 1) {
-      return ticks[ticks.length - 1];
+    return numberToFormat(`${selectPool.maxPrice || 0}`, 4);
+  }, [selectPool.maxPrice, selectPool.selectedFullRange]);
+
+  const onClickTabItem = useCallback((symbol: string) => {
+    const compareToken = tokenA.symbol === symbol ? tokenA : tokenB;
+    selectPool.setCompareToken(compareToken);
+  }, [selectPool, tokenA, tokenB]);
+
+  const selectFullRange = useCallback(() => {
+    selectPool.setFullRange(true);
+  }, [selectPool]);
+
+  function initPriceRange() {
+    if (selectPool.currentPrice && selectPool.feeTier && priceRangeType) {
+      let minRate = 0.5;
+      let maxRate = 1.5;
+      if (priceRangeType === "Active") {
+        if (selectPool.feeTier === "FEE_100") {
+          minRate = 0.995;
+          maxRate = 1.005;
+        } else {
+          minRate = 0.9;
+          maxRate = 1.1;
+        }
+      } else if (priceRangeType === "Passive") {
+        if (selectPool.feeTier === "FEE_100") {
+          minRate = 0.99;
+          maxRate = 0.01;
+        } else {
+          minRate = 0.5;
+          maxRate = 2;
```

Comment:
i'm not sure in future this value will be change? can we add this as a constant value?.  Min Rate, MaxRate

Reply:
Thanks for the suggestion. I think it would be better to manage it as a constant variable.